packages/discord.js/src/structures/Guild.js

```diff
@@ -1138,6 +1138,22 @@ class Guild extends AnonymousGuild {
     return this;
   }
 
+  /**
+   * Sets the guild's MFA level
+   * @param {GuildMFALevel} level The MFA level
+   * @param {string} reason Reason for changing the guild's MFA level
```

Comment:
The parameter should be optional:
```suggestion
   * @param {string} [reason] Reason for changing the guild's MFA level
```